src/app/core/campaign/campaign.service.ts

```diff
@@ -21,14 +21,16 @@ export class CampaignService {
 
   loadCampaignZoomFlights(id: number): Observable<{ campaignDoc: HalDoc; flightDocs: HalDoc[] }> {
     return this.augury.follow('prx:campaign', { id, zoom: 'prx:flights' }).pipe(
-      switchMap(campaignDoc => campaignDoc.follow('prx:flights').pipe(map(flightDocs => ({ campaignDoc, flightDocs })))),
-      switchMap(({ campaignDoc, flightDocs }: { campaignDoc: HalDoc; flightDocs: HalDoc }) => {
+      switchMap(campaignDoc => campaignDoc.followItems('prx:flights').pipe(map(flightDocs => ({ campaignDoc, flightDocs })))),
```

Comment:
Bit into the weeds of `HalDoc` ... but to get back an array of flight docs, you have to `followItems` which is a couple of chained `follow('prx:flights').followItems('prx:items')` to get the nested array.

So now this is correctly getting the array - but how to get the total/count that was [a few levels above](http://cav.is/lIUt4aj.png)?

Those attributes actually get set onto each of these `HalDocs` returned.  So if we got > 0 back, we can just check the first for the `.total()`.